avaframe/runComparisonModules.py

```diff
@@ -26,10 +25,10 @@
 cfgMain = cfgUtils.getGeneralConfig()
 
 # load all benchmark info as dictionaries from description files
-testList = ['avaKot']#'avaInclinedPlane', 'avaParabola', 'avaHelix', 'avaHelixChannel', 'avaWog', 'avaKot']
+testList = ['avaInclinedPlane', 'avaHelixChannel', 'avaAlr1', 'avaWog1', 'avaKot1', 'avaHit1', 'avaGar1', 'avaMal1']
```

Comment:
What are the `1`s for? Should this be in the master?

Reply:
dammit, forgot to change this... on my PC the 1 is the dem with 5m csz